src/styles/blocks/content.scss

```diff
@@ -0,0 +1,28 @@
+.content {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $width-of-content;
+  margin: 0 auto;
+  padding: 50px 0 73px;
+
+  &__card.card {
+    display: none;
+    margin-right: $marg-right;
+    margin-bottom: 46px;
+
+    &:nth-child(#{$card-in-row}n) {
+      margin-right: 0;
+    }
```

Comment:
In BEM we don`t use class nesting, and I don't understand the reason for "display: none".
You can make your styles rules for this selector little bit shorter:
instead of 
margin-right: $marg-right;
 &:nth-child(#{$card-in-row}n) {
    margin-right: 0;
Use &:not(nth-child(#{$card-in-row}n)){
   margin-right: $marg-right;
}